changelogs/config.yaml

```
@@ -0,0 +1,28 @@
+changelog_filename_template: CHANGELOG.rst
+changelog_filename_version_depth: 0
+changes_file: changelog.yaml
+changes_format: combined
+keep_fragments: false
+mention_ancestor: true
+new_plugins_after_name: removed_features
+notesdir: fragments
+prelude_section_name: release_summary
+prelude_section_title: Release Summary
+sections:
+- - major_changes
+  - Major Changes
+- - minor_changes
+  - Minor Changes
+- - breaking_changes
+  - Breaking Changes / Porting Guide
+- - deprecated_features
+  - Deprecated Features
+- - removed_features
+  - Removed Features (previously deprecated)
+- - security_fixes
+  - Security Fixes
+- - bugfixes
+  - Bugfixes
+- - known_issues
+  - Known Issues
```

```suggestion
- - major_changes
  - Major Changes
- - minor_changes
  - Minor Changes
- - breaking_changes
  - Breaking Changes / Porting Guide
- - deprecated_features
  - Deprecated Features
- - removed_features
  - Removed Features (previously deprecated)
- - security_fixes
  - Security Fixes
- - bugfixes
  - Bugfixes
- - known_issues
  - Known Issues
```
i haven't seen in the documentation few of them, why are they here:)?

---

These come from a docs meeting: https://meetbot.fedoraproject.org/ansible-docs/2020-04-28/docs_working_group_aka_dawgs.2020-04-28-14.31.html

---

Essentially a porting guide for collections consists of breaking_changes, removed_features and deprecated_features. And security_fixes was added because usually security fixes somehow get lost in the list of "regular" bugfixes.

---

would be nice to see all the options in the changelog fragment documentation ASAP